lib/internal/Magento/Framework/Image/Adapter/AbstractAdapter.php

```diff
@@ -558,6 +562,9 @@ protected function _adaptResizeValues($frameWidth, $frameHeight)
             }
         }
 
+        $frameWidth = (float)$frameWidth;
+        $frameHeight = (float)$frameHeight;
```

Comment:
Please let us know the significance of this line because both the parameters have been passed to `_checkAspectRatio` in the next line which accepts parameters as `int`

Reply:
Yes, this is correct, but they are used below the checkAspectRatio (lines 573 and below) for calculation. For more precised calculation it's better to have original/not rounded values.